### DISCUSSION TITLE
Artery ISP No Next Button

### DISCUSSION THREAD

**Discussion by A-little-confused** [keirf/flashfloppy#919] [General]
Hello,
I have a Gotek with Artery Chip,  but whilst using the programming tool i have no buttons to progress. This is on both V2.0.05 and V2.0.14.
![Error Artery](https://github.com/user-attachments/assets/fc03a684-4f42-43a6-962b-1651015228ea)

I have installed the driver and the jumper is placed as shown in the diagram on the guide you have kindly supplied. The Gotek is shown on the list when i change from USART to USB option in drop down box.

Any ideas?

**Comment by keirf** [comment]
The window has a scroll bar. Scroll down 😅

**Comment by A-little-confused** [comment]
Thank you. After feeling foolish and checking you were correct I'd just not scrolled down.

With great shame Thank you

**Comment by keirf** [comment]
No problem, sometimes we miss the obvious.

**Comment by C128Guy** [comment]
I just had a similar issue, but there was no way to scroll.  I had to hookup a larger resolution external display to my laptop for it to work and scroll down.
